data/nation/texan.ruleset

```diff
@@ -5,28 +5,35 @@ plural=_("?plural:Texans")
 groups="American"
 legend=_("Texas, \"the Lone Star State\" is the second largest and second\
  most populous state of the USA, famous for its \"larger than life\"\
- cowboy mentality. During its history, Texas has been ruled in full or in\
- part by six countries: France, Spain, Mexico, the Texas Republic, the\
- Confederacy and the USA - as expressed in the slogan \"six flags over\
- Texas\".")
+ cowboy culture and Spanish heritage. After the famous Battle of the Alamo de Bexar,\
+ The Texians defeated the Mexicans at San Jacinto and declared independence.\
+ The Republic of Texas lasted from 1836 to 1846, after which it became the\
+ 28th state of the United States. Historically, Texas was ruled in full or in\
+ part by six nations: France, Spain, Mexico, the Republic of Texas, the\
+ Confederacy, and the USA, as expressed in the slogan \"six flags over Texas\".")
 
 leaders = {
  "name",                "sex"
- "Stephen F. Austin",   "Male"
- "Juan N. Seguin",      "Male"
- "Sam Houston",         "Male"
- "Mirabeau B. Lamar",   "Male"
- "James P. Henderson",  "Male"
- "Ma Ferguson",         "Female"
- "John Connally",       "Male"
- "Lyndon B. Johnson",   "Male"
- "George W. Bush",      "Male"
+ "Stephen F. Austin",      "Male"
+ "Juan N. Seguin",         "Male"
+ "Sam Houston",            "Male"
+ "Francis Lubbock",        "Male"
+ "Mirabeau B. Lamar",      "Male"
+ "James P. Henderson",     "Male"
+ "Lawrence Sullivan Ross", "Male"
+ "Ma Ferguson",            "Female"
+ "John Connally",          "Male"
+ "Lyndon B. Johnson",      "Male"
+ "Ann Richards",           "Female"
+ "George W. Bush",         "Male"
 }
 
 ruler_titles = {
  "government",      "male_title",      "female_title"
+ "Republic",        _("President %s"), _("?female:President %s")
+ "Democracy",       _("Governor %s"),  _("?female:Governor %s")
```

Comment:
Maybe also use "governor" for Monarchy, since it was already the name in the Spanish era?